test/measureOperationsUtils.test.js

```diff
@@ -18,4 +22,113 @@ describe('measureOperationsUtils functions', () => {
     const REQUIRED_PARAMS = ['test', 'test2'];
     checkRequiredParams(req, REQUIRED_PARAMS, 'test');
   });
+
```

Comment:
I would add a test here as well that validateEvalMeasureParams doesn't throw an error when passed valid parameters